t/Makefile.am

```diff
@@ -57,27 +57,30 @@ TESTS = Test01-check_programs_exist.sh \
         Test25-hydrology.sh \
         Test26-reference_fregrid_gcc.sh \
         Test27-multiple_nests.sh \
-        01-make_hgrid.sh \
+        Test28-fregrid_stretched.sh \
+        Test29-fregrid_no_stretched.sh \
```

Comment:
You've added two tests here to run (Test28 and Test29) but included in the PR are other tests, up to Test37. Does Test28 and 29 contain everything now? If so, can the others be removed?

Reply:
Yes Chris. The Test28 and Test29 contain everything now.  Those other one can be removed.